### PR TITLE
7.0 - Fix Combatant struct

### PR DESCRIPTION
**Pull request by valarnin**
This needs more testing. My brain's fried, so I'm not even sure if I tested all the properties.

Also finally dropped `MonsterType` and `AggressionStatus` since we haven't tracked them down in like an entire expansion.

### CODE REVIEW

File: OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory70.cs
Comment (wexxlee) [inline OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory70.cs:204]: Tested and this seems to be right (or at least is returning the expected data).

**Review by wexxlee — APPROVED** [review]
Gave this a workout and seems to be fine to me.  Defer to you if you want to test more before merging or merge as-is and see if anything crops up later.

**Comment by valarnin** [comment]
The ID is in all three offsets. Needs tested for e.g. enemy combatants to
ensure that we have the correct field that only gets set for player actors.

**Pull request merged**
